packages/design-system/src/components/Drawer/Drawer.jsx

```diff
@@ -49,41 +49,37 @@ export class SlidingPanel extends React.PureComponent {
     const Heading = `h${this.props.headingLevel}` || `h3`;
 
     /* eslint-disable jsx-a11y/no-noninteractive-tabindex, react/no-danger */
-    const slidingPanelMarkup = () => (
-      <div
-        aria-labelledby={this.id}
-        className={classNames(className, 'ds-c-sliding-panel')}
-        role="dialog"
-      >
-        <div className="ds-c-sliding-panel__window">
-          <div className="ds-c-sliding-panel__header">
+    const drawerMarkup = () => (
```

Comment:
Not related to the name change, so out of scope, but I just noticed this function and that it could be flattened to just be a value instead of a function that gets called later, since it is always called in every logical branch.

Reply:
You're right. I created a separate PR here #1224